3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentRegistry.scala

```diff
@@ -157,7 +159,16 @@ case class EnrichmentRegistry(private val configs: EnrichmentMap) {
    */
   def getAnonIpEnrichment: Option[AnonIpEnrichment] =
     getEnrichment[AnonIpEnrichment]("anon_ip")
-
+  
+  /**
+   * Returns an Option boxing the UaParserUtilityConfigEnrichment
+   * config value if present, or None if not
+   * 
+   * @return Option boxing the UaParserUtilityConfigEnrichment instance
+   */
+  def getUaParserUtilityConfigEnrichment: Option[UaParserUtilityConfigEnrichment] =
+    getEnrichment[UaParserUtilityConfigEnrichment]("user_agent_utils_config")
```

Comment:
The argument is the name of the enrichment, so this should be `...]("user_agent_utils")`